.github/workflows/codecov.yml

```diff
@@ -14,7 +16,11 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - uses: taiki-e/install-action@cargo-llvm-cov
-      - run: cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
+      - shell: bash
+        run: |
+          cargo llvm-cov \
```

Comment:
Can you add a comment here to explain how you arrived at the list of features, and a comment on the `[features]` sections in the relevant `Cargo.toml` files with a reminder to keep this in sync?

Reply:
Should I include my `cargo metadata` invocation in the comment in this file or no?

Reply:
No, I don't think that's necessary -- making changes should mostly be pretty straightforward, I guess?

Reply:
What I mean is, you said I should "explain how you arrived at the list of features", and that invocation is how I did it.

So I'm wondering what comment you're looking for in this particular file?

Reply:
Ah, sure, let's include your `cargo metadata` invocation as a comment.

Reply:
All done, let me know if this is what you wanted!

Reply:
This looks fine to me.